tests/fixtures/exception_groups.py

```diff
@@ -0,0 +1,48 @@
+# flake8: noqa
+
+# this creates an exception with a very small __traceback__, so it's easier to
+# assert against in tests
+def generate_exception(exception_class, message):
+    try:
+        raise exception_class(message)
+    except BaseException as exception:
+        return exception
+
+
+def raise_exception_group_with_no_cause():
+    raise ExceptionGroup(
+        'the message of the group',
+        [
+            generate_exception(Exception, 'exception #1'),
+            generate_exception(ArithmeticError, 'exception #2'),
+            generate_exception(NameError, 'exception #3'),
+            generate_exception(AssertionError, 'exception #4'),
+        ]
+    )
+
+
+try:
+    raise_exception_group_with_no_cause()
+except BaseExceptionGroup as exception_group:
+    exception_group_with_no_cause = exception_group
+
+
+class MyExceptionGroup(BaseExceptionGroup):
+    pass
+
+
+def raise_base_exception_group_subclass_with_no_cause():
+    raise MyExceptionGroup(
+        'my very easy method just speeds up (n)making exception groups',
```

Comment:
> (n)making

Not sure what you mean by this? Maybe a term I've not heard before...

Reply:
`MyExceptionGroup` made me think of this planet mnemonic:

> My Very Easy Method Just Speeds Up Naming Planets

but there are no synonyms for 'making' that start with 'n' ☹️